dfusion_rust_core/src/models/account_state.rs

```diff
@@ -214,26 +214,27 @@ pub mod test_util {
 pub mod tests {
     use super::*;
     use crate::models::order::BatchInformation;
-    use crate::models::TOKENS;
     use std::str::FromStr;
     use web3::types::{Bytes, H256};
 
+    const NUM_TOKENS: u16 = 10;
```

Comment:
Why 10? Is there a specific reason for that or is it just an arbitrary amount?

Reply:
Just arbitrary for the unit tests, but I will probably remove this. Still need to make a helper function that maps u16 to bytes (i.e. Vec<u8>)

Reply:
I have included a comment that this is not super relevant, just used for readability throughout the unit tests. Also set as a constant within the tests of some other files too. Hope this is alright,.